static/sass/_snapcraft_market_screenshots.scss

```diff
@@ -1,7 +1,10 @@
 @mixin snapcraft-market-screenshots {
   .p-listing-images {
-    justify-content: flex-start;
-    margin-right: -$spv-inner--small;
+    display: grid;
```

Comment:
I think we shouldn't do that.

Removing custom implementation to replace it with Vanilla grid classes (a little bit customized) how we do it in other places is fine, but here we are implementing just another custom styling that seems much more complicated than it was before. I don't think it's worth it.
Especially that it behaves bit weirdly during reordering with drag and drop (spacing between items is off when dragging, especially when they wrap in 2 lines on smaller screens).

Reply:
If it turns out this is the last place we need out "fluid grid" pattern, let's move any needed fluid grid styles here and remove the `p-fluid-grid`, but I wouldn't rewrite the whole thing in grid just for the sake of it.

Unless we can simply work this out with Vanilla col classes, which apparently is not that straightforward in this case.